shared/js/background/chrome-events.es6.js

```diff
@@ -308,6 +308,19 @@ const browserWrapper = require('./chrome-wrapper.es6')
 chrome.runtime.onMessage.addListener((req, sender, res) => {
     if (sender.id !== chrome.runtime.id) return
 
+    if (req.registeredContentScript) {
+        const argumentsObject = getArgumentsObject(sender.tab.id)
+        if (argumentsObject.site?.isBroken) {
+            console.log('temporarily skip fingerprint protection for site: ' + sender.tab.url +
+        'more info: https://github.com/duckduckgo/content-blocking-whitelist')
+            return
+        }
+        if (!argumentsObject.site.whitelisted) {
```

Comment:
What is the difference between `isBroken` and `whitelisted`?.

Do we also need the optional chaining on `site` here? If not, we can also remove it in the above if statement.

EDIT: Looking at `getArgumentsObject` it looks like `site` will always be an object.

Reply:
whitelisted is the existing user disabling, isBroken comes from the list